runtime/manager_test.go

```diff
@@ -0,0 +1,365 @@
+package runtime
+
+import (
+	"io"
+	"math/big"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/benbjohnson/clock"
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/runtime/genesis"
+	"github.com/pokt-network/pocket/shared/converters"
+	"github.com/pokt-network/pocket/shared/core/types"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func TestNewManagerFromReaders(t *testing.T) {
+	type args struct {
+		configReader  io.Reader
+		genesisReader io.Reader
+		options       []func(*Manager)
+	}
+
+	buildConfigBytes, err := os.ReadFile("../build/config/config1.json")
```

Comment:
NIT: Please move this path (and the one below) into constants at the top of the file with `Default` prefix